ostd/src/boot/kcmdline.rs

```diff
@@ -159,6 +171,21 @@ impl From<&str> for KCmdlineArg {
                         }
                         result.initproc.path = Some(value.to_string());
                     }
+                    "loglevel" => {
+                        if result.log_level.is_some() {
+                            panic!("Loglevel assigned twice in the command line!");
+                        }
+                        result.log_level = Some(match value {
+                            "0" => Level::Error,
```

Comment:
These parameters are passed as strings. So what is the point of specifying them as integers? Apparently, specifying the log level with `loglevel=error` is more user friendly than `loglevel=0`. And the former can be implemented as easily as the latter.

Reply:
I prefer using strings for input, but I also considered Linux kernel parameters and decided to follow the Linux kernel parameter.

I will change the log level to string in the new version.